Corona-Warn-App/src/main/res/layout/analytics_ppa_userinfo_input_fragment.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Is it still required to have this tag? if `ViewBinding` is enough we should get rid of it